emily_landi/.gitignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
I think the .gitignore needs to be in the directory above this (should be in the root directory--the folder that's made when you run "git clone".) The node_modules folder was still added. If you remove it, I'll merge your pull request.